src/js/components/List/List.js

```diff
@@ -192,7 +193,14 @@ const List = React.forwardRef(
                     adjustedEvent.item = item;
                     adjustedEvent.index = index;
                     onClickItem(adjustedEvent);
+                    // put focus back on the List container to meet
+                    // WCAG accessibility guidelines that focus remains
+                    // on `ul`
+                    (ref || listRef).current.focus();
                   },
+                  // prevent default focus outline from being shown
+                  // onMouseDown since `ul` is maintaining focus
+                  onMouseDown: event => event.preventDefault(),
                   onMouseOver: () => setActive(index),
                   onMouseOut: () => setActive(undefined),
                   onFocus: () => setActive(index),
```

Comment:
I wonder if we should do something here, so that focus can't go on the item at all.

Reply:
Agreed, that was what I was attempting to do with line 203. I couldn't figure out another way to avoid focus being put on it. Do you have any immediate thoughts?

Reply:
I was suggesting that we don't prevent it from getting focus, but we steal it back as soon as it takes it.

Reply:
I see -- I tried applying the same logic as was applied with `onClick` which is `listRef.current.focus();` but it doesn't seem to be having the same effect on this event. Will need to dive in a little more.

Reply:
This is resolved by instead grabbing focus back in the `onFocus` handler instead of touching `onMouseDown`. thank you, @ericsoderberghp!